app/views/SelectedUsersView/Header.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useRef } from 'react';
+import { FlatList, View, Text } from 'react-native';
+
+import { themes } from '../../lib/constants';
+import SearchBox from '../../containers/SearchBox';
+import I18n from '../../i18n';
+import { ISelectedUser } from '../../reducers/selectedUsers';
+import { useTheme } from '../../theme';
+import sharedStyles from '../Styles';
+import { useAppSelector } from '../../lib/hooks';
+import Chip from '../../containers/Chip';
+
+const Header = ({
+	onChangeText,
+	useRealName,
+	onPressItem
+}: {
+	useRealName: boolean;
+	onChangeText: (text: string) => void;
+	onPressItem: (userItem: ISelectedUser) => void;
+}) => {
+	const flatlist = useRef<FlatList>();
+	const { theme } = useTheme();
+	const { users } = useAppSelector(state => ({
+		users: state.selectedUsers.users
+	}));
+
+	const onContentSizeChange = () => flatlist?.current?.scrollToEnd({ animated: true });
+
+	return (
+		<View style={{ backgroundColor: themes[theme].backgroundColor }}>
+			<SearchBox onChangeText={(text: string) => onChangeText(text)} testID='select-users-view-search' />
+			{users.length === 0 ? null : (
+				<View>
+					<Text style={{ ...sharedStyles.textRegular, color: themes[theme].auxiliaryTintColor, marginLeft: 16 }}>
+						{I18n.t('N_Selected_members', { n: users.length })}
+					</Text>
+					<FlatList
+						data={users}
+						ref={(ref: FlatList) => (flatlist.current = ref)}
+						onContentSizeChange={onContentSizeChange}
+						keyExtractor={item => item._id}
+						renderItem={({ item }) => {
+							const name = useRealName && item.fname ? item.fname : item.name;
+							const username = item.search ? (item.username as string) : item.name;
+
+							return (
+								<Chip text={name} avatar={username} onPress={() => onPressItem(item)} testID={`selected-user-${item.name}`} />
+							);
+						}}
+						keyboardShouldPersistTaps='always'
+						contentContainerStyle={{ paddingLeft: 16 }}
```

Comment:
paddingHorizontal. When you reach the end of the list, it's expected to have a padding at the end.